### PR TITLE
DCS-1060 extending VLB update to accept changing courtId

### PR DESCRIPTION
**Pull request by louisenorris**

### CODE REVIEW

File: src/main/java/uk/gov/justice/digital/hmpps/whereabouts/dto/VideoLinkBookingUpdateSpecification.kt [hunk 1]
```diff
@@ -5,6 +5,14 @@ import javax.validation.Valid
 
 data class VideoLinkBookingUpdateSpecification(
 
+  @ApiModelProperty(
+    value = "The identifier of the court that requires the appointment.  This must be one of the court identifier values from the courts register service.",
+    example = "CMBGMC",
+    required = false,
+    notes = "One of court or courtId must contain a value.  courtId takes precedence over court."
```
Comment (andrewrlee) [inline src/main/java/uk/gov/justice/digital/hmpps/whereabouts/dto/VideoLinkBookingUpdateSpecification.kt:12]: Is this right? Isn't it always going to be courtId?

Comment (louisenorris) [reply src/main/java/uk/gov/justice/digital/hmpps/whereabouts/dto/VideoLinkBookingUpdateSpecification.kt:12]: It is but wasn't that the breaking change we were talking about which we could update once BVL has been released if we decide to?

Comment (louisenorris) [reply src/main/java/uk/gov/justice/digital/hmpps/whereabouts/dto/VideoLinkBookingUpdateSpecification.kt]: Updated and notes removed.

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**